website/content/docs/platform/k8s/vso/examples.mdx

```diff
@@ -11,6 +11,34 @@ The Operator project provides the following examples:
 - Sample use-cases are documented [here](https://github.com/hashicorp/vault-secrets-operator#samples)
 - A Terraform based demo can be found [here](https://github.com/hashicorp/vault-secrets-operator/tree/main/demo)
 
+## Configuring the JWT auth method to avoid API calls to the Kubernetes API Server
```

Comment:
I think one of the major benefits of this approach is that the user does not have to expose the K8s API server to the world. The title seems to include that info, so I wonder if it could make sense to expand on it here.

Reply:
Added a comment in 6ba9d74